bin/sandbox

```diff
@@ -88,12 +79,7 @@ fi
 
 unbundled bundle install --gemfile Gemfile
 unbundled bin/rails db:drop db:create
-unbundled bin/rails generate solidus:install \
-  --auto-accept \
```

Comment:
I'm not super happy removing `auto-accept` by default, as it slows the development feedback. What do you think about keeping it?

Reply:
I think it's no big deal either way, easy to add or remove locally as needed, so I'll add it back (I had to remove it in order to test out the interactive `solidus:install` wizard).